.travis.yml

```diff
@@ -1,3 +1,7 @@
 language: node_js
 node_js:
-  - 0.10
+  - "0.10"
+  - "0.12"
+  - "iojs"
+install:
+  - npm install
```

Comment:
Just a reminder that it's not necessary to specify `npm install` as an install command, because Travis does this automatically based on the fact that a `package.json` exists.